datadog/resource_datadog_dashboard_timeseries_test.go

```diff
@@ -6,253 +6,197 @@ import (
+//                              "facet":"@duration",
+//                              "aggregation":"avg",
+//                              "order":"desc"
+//                           },
+//                           "limit":10
 //                        }
-//                    },
-//                    {
-//                        "style": {
-//                            "line_width": "thin",
-//                            "palette": "green",
-//                            "line_type": "dotted"
-//                        },
-//                        "security_query": {
-//                            "index": "*",
-//                            "search": {
-//                                "query": ""
-//                            },
-//                            "group_by": [
-//                                {
-//                                    "facet": "service",
-//                                    "sort": {
-//                                        "facet": "status",
-//                                        "aggregation": "cardinality",
-//                                        "order": "desc"
-//                                    },
-//                                    "limit": 10
-//                                }
-//                            ],
-//                            "compute": {
-//                                "facet": "status",
-//                                "aggregation": "cardinality"
-//                            }
-//                        },
-//                        "display_type": "bars"
-//                    }
-//                ],
-//                "time": {
-//                    "live_span": "5m"
-//                },
-//                "title": "system.cpu.user, env, process.stat.cpu.total_pct.norm, network.bytes_read, @d...",
-//                "legend_size": "2",
-//                "type": "timeseries",
-//                "events": [
-//                    {
-//                        "q": "sources:test tags:1",
-//                        "tags_execution": "and"
-//                    }
-//                ]
-//            },
-//            "layout": {
-//                "y": 2,
-//                "x": 1,
-//                "height": 15,
-//                "width": 47
-//            },
-//            "id": 0
-//        }
-//    ],
-//    "layout_type": "free"
+//                     ],
+//                     "compute":{
+//                        "facet":"@duration",
+//                        "interval":10,
+//                        "aggregation":"avg"
+//                     }
+//                  },
+//                  "display_type":"area"
+//               }
+//            ],
+//            "legend_size":"2",
+//            "type":"timeseries",
+//            "events":[
+//               {
+//                  "q":"sources:test tags:1",
+//                  "tags_execution":"and"
+//               }
+//            ]
+//         },
+//         "id":203606066001657
```

Comment:
Since we redact most ids/urls/handles, we should probably redact these widget ids